src/components/Navbar.js

```diff
@@ -121,7 +121,12 @@ Navbar.propTypes = {
   logoutUserAction: PropTypes.func,
 };
 
-const mapStateToProps = ({ user }) => ({ isLoggedIn: !!user && !!user.handle });
+const mapStateToProps = ({ user }) => {
+  const isLoggedIn = !!user && !!user.userInfo.handle;
```

Comment:
you're not null-checking userInfo here
add `&& !!user.userinfo`

Reply:
Fixed